### ISSUE TITLE
Bringing project to Apache

### ISSUE THREAD

**Issue by lewismc** [joshua-decoder/joshua_translation_engine#12]
Hi Folks,
I feel that there would be more chance of increased development of this codebase if it were donated to the Apache Joshua (Incubating) project under a [SGA](https://www.apache.org/licenses/software-grant.txt).
Any thoughts on this?
Thank in advance.

**Comment by lewismc** [comment]
@mjpost any thoughts on this? Do you think it is worth it? My thoughts are that it would be really beneficial and I also think that the project would get more attention if it is/was hosted alongside Joshua source @apache. wdyt?

**Comment by mjpost** [comment]
Yes, we should pull it in. It would fit great inside Joshua on the 7 branch. I'll do that once we get the 6.1 release out of the way.

**Comment by lewismc** [comment]
Dynamite. Once we do I'll close this off. Thanks @mjpost
